File: 2 semestre/Laboratório de Desenvolvimento de Algoritmos/Lista de Exercícios 4/src/Ex13.java
//Importando Scanner
import java.util.Scanner;

public class Ex13 {

	public static void main(String[] args){
		
		//Instanciando Scanner
		Scanner ent = new Scanner(System.in);
		
		System.out.println("\n\n");
		
		//vars
		int v[] = new int[10];
		int i, j;
		
		//Entrada
		for(i=0; i<10; i++)
		{
			System.out.print("Digite v[" + i + "]: ");
			v[i] = ent.nextInt();
		}
		
		//Processamento e Saída
		System.out.println("\nValores iguais:");
		for(i=0; i<10; i++)
		{
			for(j=i+1; j<10; j++)
			{
				if(v[i] == v[j])
					System.out.println(v[i]);
			}
		}
		
		
	}
}

File: 3 semestre/Exercícios Soltos/Estrutura de Dados I/Listas/Lista de Matrizes/ex13.c
#include <stdio.h>

main()
{
    /* vars */
    int a[5][5], b[5][5], diferenca[5][5], i, j;

    /* Entrada */
    for (i = 0; i < 5; i++) {
        for (j = 0; j < 5; j++) {
            printf("Digite A[%d][%d]: ", i, j);
            scanf("%d", &a[i][j]);
        }
    }

    for (i = 0; i < 5; i++) {
        for (j = 0; j < 5; j++) {
            printf("Digite B[%d][%d]: ", i, j);
            scanf("%d", &b[i][j]);
        }
    }

    /* Processamento e Saída */
    for (i = 0; i < 4; i++) {
        for (j = 0 ; j < 4; j++) {
            diferenca[i][j] = a[i][j] - b[i][j];
            printf("%d  ", diferenca[i][j]);
        }
        printf("\n");
    }
}
